py/server/deephaven/table.py

```diff
@@ -537,6 +539,15 @@ def is_refreshing(self) -> bool:
             self._is_refreshing = self.j_table.isRefreshing()
         return self._is_refreshing
 
+    @property
+    def update_graph(self) -> _JUpdateGraph:
```

Comment:
Two problems:
1) The API is exposing Java objects to the user.
2) The type hint is referring to a private variable.